src/app/market/sell/add-item/add-item.component.html

```diff
@@ -8,8 +8,8 @@
         <mat-icon class="icon" fontSet="partIcon" fontIcon="part-select"></mat-icon>
 
         <!--hide input -->
-        <input style="display: none;" type="file"
-               accept="image/*" multiple="true" id="fileInput">
+        <!-- <input style="display: none;" type="file"
```

Comment:
NO!

You are again breaking this, what the heck...

Reply:
It breaks the test by the way.

Other than that - this PR is mergeable.

Reply:
@kewde if it is display:none then how it breaks the component tho ?

Thanks

Reply:
tho I have changed that to class rather commenting it out 

Thanks